tests/integration_tests/functional/test_api.py

```diff
@@ -125,8 +136,9 @@ def test_api_put_update_pre_boot(test_microvm_with_api):
     mem_size_mib = microvm_config_json['mem_size_mib']
     assert response_json['mem_size_mib'] == mem_size_mib
 
-    cpu_template = str(microvm_config_json['cpu_template'])
-    assert response_json['cpu_template'] == cpu_template
+    if platform.machine() != 'aarch64':
```

Comment:
Nit: I would rather check if `platform.machine() == 'x86_64'` here. Since we only support templates for x86_64.